GI01SUMO/GI01SUMO_D00001_ingest.csv

```diff
@@ -24,12 +24,12 @@ mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_telemetered_driver,/omc_da
 mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00001/cg_data/dcl11/imm/ctdmo10/ctdmo*.DAT,GI01SUMO-RII11-02-CTDMOR018,telemetered,Available,Inductive modem 20
 mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00001/cg_data/dcl11/imm/ctdmo11/ctdmo*.DAT,GI01SUMO-RII11-02-CTDMOR019,telemetered,Available,Inductive modem 21
 mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00001/cg_data/dcl11/imm/ctdmo12/ctdmo*.DAT,GI01SUMO-RII11-02-CTDMOR020,telemetered,Available,Inductive modem 22
-#mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_telemetered_driver,,GI01SUMO-RII11-02-DOSTAD031,telemetered,Not Deployed,Not deployed in this instance
-#mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_telemetered_driver,,GI01SUMO-RII11-02-DOSTAD032,telemetered,Not Deployed,Not deployed in this instance
-#mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_telemetered_driver,,GI01SUMO-RII11-02-DOSTAD033,telemetered,Not Deployed,Not deployed in this instance
-#,,GI01SUMO-RII11-02-FLORDG031,telemetered,Not Deployed,Not deployed in this instance
-#,,GI01SUMO-RII11-02-FLORDG032,telemetered,Not Deployed,Not deployed in this instance
-#,,GI01SUMO-RII11-02-FLORDG033,telemetered,Not Deployed,Not deployed in this instance
+#mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00001/cg_data/dcl11/imm/ctdbp01/ctdbp*.DAT,GI01SUMO-RII11-02-DOSTAD031,telemetered,Not Deployed,Not deployed in this instance
+#mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00001/cg_data/dcl11/imm/ctdbp02/ctdbp*.DAT,GI01SUMO-RII11-02-DOSTAD032,telemetered,Not Deployed,Not deployed in this instance
+#mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_telemetered_driver,/omc_data/whoi/OMC/GI01SUMO/D00001/cg_data/dcl11/imm/ctdbp03/ctdbp*.DAT,GI01SUMO-RII11-02-DOSTAD033,telemetered,Not Deployed,Not deployed in this instance
```

Comment:
I think these should be: mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_telemetered_driver